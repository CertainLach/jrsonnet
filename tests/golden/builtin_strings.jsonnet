local str = 'ab::cd::ef';
{
	split: std.split(str, '::'),
	splitlimit: std.splitLimit(str, '::', 1),
	splitlimitRNoLimit: std.splitLimit(str, '::', -1),
	splitlimitR: std.splitLimitR(str, '::', 1),
}
