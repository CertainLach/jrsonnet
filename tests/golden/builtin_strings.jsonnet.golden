{
    "split": [
        "ab",
        "cd",
        "ef"
    ],
    "splitlimit": [
        "ab",
        "cd::ef"
    ],
    "splitlimitR": [
        "ab::cd",
        "ef"
    ],
    "splitlimitRNoLimit": [
        "ab",
        "cd",
        "ef"
    ]
}